iib/workers/tasks/build_regenerate_bundle.py

```diff
@@ -152,37 +152,15 @@ def _apply_package_name_suffix(metadata_path, organization=None):
     :return: a tuple with the package name and a dictionary of labels to set on the bundle.
     :rtype: tuple(str, dict)
     """
-    annotations_yaml_path = os.path.join(metadata_path, 'annotations.yaml')
-    if not os.path.exists(annotations_yaml_path):
-        raise IIBError('metadata/annotations.yaml does not exist in the bundle')
-
-    with open(annotations_yaml_path, 'r') as f:
-        try:
-            annotations_yaml = yaml.load(f)
-        except ruamel.yaml.YAMLError:
-            error = 'metadata/annotations/yaml is not valid YAML'
-            log.exception(error)
-            raise IIBError(error)
-
-    if not isinstance(annotations_yaml.get('annotations', {}), dict):
-        raise IIBError('The value of metadata/annotations.yaml must be a dictionary')
-
+    annotations_yaml = _get_package_annotations(metadata_path)
     package_label = 'operators.operatorframework.io.bundle.package.v1'
-    package_annotation = annotations_yaml.get('annotations', {}).get(package_label)
-    if not package_annotation:
-        raise IIBError(f'{package_label} is not set in metadata/annotations.yaml')
-
-    if not isinstance(package_annotation, str):
-        raise IIBError(f'The value of {package_label} in metadata/annotations.yaml is not a string')
+    package_annotation = annotations_yaml['annotations'][package_label]
```

Comment:
I don't see this behavior in the docstring. Could you please add it?

Without an explanation, it's confusing that `organization` can be `None`.